extensions/cornerstone/src/commandsModule.ts

```diff
@@ -19,6 +19,9 @@ import toggleStackImageSync from './utils/stackSync/toggleStackImageSync';
 import { getFirstAnnotationSelected } from './utils/measurementServiceMappings/utils/selection';
 import getActiveViewportEnabledElement from './utils/getActiveViewportEnabledElement';
 import { CornerstoneServices } from './types';
+import { split } from 'lodash';
```

Comment:
aviod using lodash if possible, I don't think we need to rely on lodash for split

Reply:
Removed